src/deserialize/index.spec.js

```diff
@@ -136,4 +240,10 @@ describe('deserialize', () => {
     expect(resp).toEqual(resp);
     expect(result).toEqual(expectedArrayResponse);
   })
+  
+  it('deserializes and camel case the object keys', () => {
+    const result = deserialize(respWithSeparators, {convertsToCamelCase: true});
```

Comment:
Based on the other comments, this would end up being `const result = deserialize(respWithSeparators, { setKeyTransform: 'camelLower' });`